code/modules/clothing/clothing.dm

```diff
@@ -200,6 +200,12 @@
 	playsound(src.loc, /decl/sound_category/glass_break_sound, 70, 1)
 	qdel(src)
 
+/obj/item/clothing/proc/catch_fire()
+	return
+
+/obj/item/clothing/proc/extinguish_fire()
```

Comment:
How stupid would it be to just have these procs on /obj/item (or even /obj) so that we can have flammable anything (if we want to)?

Reply:
maybe. if someone lowers it down to that, converting it over would be really easy

Reply:
actually i think this is the one blocking the PR so i'll just do it to get it in